.circleci/config.yml

```diff
@@ -463,7 +498,7 @@ jobs:
           command: |
             aws s3 cp s3://android-circleci-payload/gradle-wrapper.jar gradle/wrapper/gradle-wrapper.jar
             echo "${ANDROID_TESTS_CREDENTIALS_JSON}"  | base64 -di   > awsconfiguration.json
```

Comment:
Can we store this in s3 like test configuration json instead of storing it as an environment variable in favor of consistency.